action.yml

```
@@ -1,5 +1,5 @@
-name: '1Password Action'
-description: 'Read items from a 1Password vault for use in your github workflow secrets'
+name: '1Password Secrets'
+description: 'Read items from a 1Password vault for use in your GitHub Action workflows'
```

```suggestion
description: 'Read items from 1Password for use in your GitHub Action workflows'
```
because it can read from 1+ vaults in the same action